### DISCUSSION TITLE
Google AuthSession not working if user's account has 2FA?

### DISCUSSION THREAD

**Discussion by ftzi** [expo/expo#16735] [Archived: Questions]
Hi, we've just released our app and we are having this issue with an user who has Two Factor auth on his Google account:

https://user-images.githubusercontent.com/28177550/159852070-57470b3f-4dd2-470e-af4f-68d1fc9d7782.mp4

*(his recording)*.

For normal users, instead of going back to the SignIn screen, it loads properly and the user is then logged.

Sorry for the low amount of info but I am quite lost with this. I am going to enable 2FA on my mobile soon to test it better, but I am already sharing this issue to see if someone has been through this already,

I am using Firebase and its related code as described in the docs: https://docs.expo.dev/guides/authentication/#web-apps

**Comment by jpschw** [comment]
Hey I found if you call startAsync like this it works:
.startAsync({ showInRecents: true });

https://docs.expo.dev/versions/latest/sdk/auth-session/#authsessionoptions

**Reply by farzd** [reply]
> Hey I found if you call startAsync like this it works: .startAsync({ showInRecents: true });
> 
> https://docs.expo.dev/versions/latest/sdk/auth-session/#authsessionoptions

this worked for me

```
 await WebBrowser.openAuthSessionAsync(
        data?.url ?? '',
        redirectTo,
        { showInRecents: true }
      )
```

**Comment by atoool** [comment]
Got the same issue. Is there any fix?
